### ISSUE TITLE
Add 'addOnBlur' attribute to Chips

### ISSUE THREAD

**Issue by mertsincan** [primefaces/primefaces#4838]
Whether to add an item when the input loses focus.

**Comment by demanzano** [comment]
I got a NPE if I do not explicity include the attrubute
addOnBlur="false"
on p:chips  !

Here it is:

23:04:56,255 SEVERE [org.primefaces.application.exceptionhandler.PrimeExceptionHandler] (default task-6) null: java.lang.NullPointerException
	at org.primefaces.component.chips.Chips.isAddOnBlur(Chips.java:127)
	at org.primefaces.component.chips.ChipsRenderer.encodeScript(ChipsRenderer.java:189)

**Comment by mertsincan** [comment]
Hi @demanzano,

I couldn't replicate it. It works fine for me;
```
<p:chips id="chips" />

Or 
<p:chips id="chips" addOnBlur="false" />
```

**Comment by mertsincan** [comment]
I think you use 6.2.21. Is it right?

**Comment by demanzano** [comment]
it happens to me regularly, my setup is: Wildfly 15, Java EE 8, PF 6.2.21, PFext 6.2.10 running on Mac.  I agree it's very strange, I don't know why...

**Comment by mertsincan** [comment]
Merge bug :( Fixed for 6.2.22. Sorry for the mistake.
https://github.com/primefaces/primefaces/issues/4878

Best Regards,

**Comment by demanzano** [comment]
Thank you very much for very quick answer & fix !!
